hack/verify-staticcheck.sh

```diff
@@ -25,6 +25,7 @@ source "${KUBE_ROOT}/hack/lib/util.sh"
 kube::golang::verify_go_version
 
 FOCUS="${1:-}"
+FOCUS="${FOCUS/%\//}" # Remove the ending "/"
```

Comment:
Emm, how about this? 
```suggestion
FOCUS="${FOCUS%/}" # Remove the ending "/"
```
Ref: http://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Shell-Parameter-Expansion
Anyway, SGTM!

Reply:
That's neat! Updated.